Update incorrect details on storage-failover-customer-managed-planned.md

DespindolaMS opened the pull request:
Delete "RA-GRS or RA-GZRS users will continue to have read access to their secondary region." and rename "primary region" to "storage account".

DespindolaMS committed:
Update incorrect details on storage-failover-customer-managed-planned.md

Delete "RA-GRS or RA-GZRS users will continue to have read access to their secondary region." and rename "primary region" to "storage account".

Padmalathas commented on articles/storage/common/storage-failover-customer-managed-planned.md:
```diff
@@ -83,7 +83,7 @@ While in the failover state, perform your disaster recovery testing.
 After testing is complete, perform another failover to failback to the original primary region. During the failover process, as shown in the following image:
 
-- The primary region temporarily loses both read and write access. RA-GRS or RA-GZRS users will continue to have read access to their secondary region.
+- The storage account temporarily loses both read and write access.
```
@DespindolaMS, thank you for the contribution. Request you to please replace the sentence with the below:
"The storage account in both primary and secondary region experiences a temporary loss of both read and write access."

And update the ms.date in the metadata section of the document to current and move forward with the sign off. Thank you!

DespindolaMS committed:
Update failover process descriptions and date

Overwrite "The storage account temporarily loses both read and write access." to "The storage account in both primary and secondary region experiences a temporary loss of both read and write access."
